pkg/storage/gcp/gcp_test.go

```diff
@@ -51,6 +51,9 @@ func getStorage(t testing.TB) *Storage {
 	bucketName := randomBucketName(os.Getenv("DRONE_PULL_REQUEST"))
 	cfg := getTestConfig(bucketName)
 	if cfg == nil {
+		// Don't fail if there's no test config, so that these tests don't
+		// fail when you run them locally
+		t.Log("No GCS Config found")
```

Comment:
the `t.Log` won't show up in stdout (or stderr) unless you pass the `-v` flag. If you pass the `-v` flag, then you'll also see that the test got skipped like so: `SKIP TestXXX` -- so this seems unnecessary unless I'm missing something

Reply:
Good point. You're not missing anything - I put this log in so that we'd see something specific about the GCS config in our drone logs, if we mess up the env var in the future